test/test_examples.py

```diff
@@ -954,12 +956,13 @@ def _insert_employee_retry_commit(session):
             run_transaction_with_retry(_insert_employee_retry_commit, session)
 
         employee = employees.find_one({"employee": 4})
+        assert employee is not None
         self.assertIsNotNone(employee)
         self.assertEqual(employee['status'], 'Active')
 
         # Start Transactions Retry Example 3
 
-        def run_transaction_with_retry(txn_func, session):
+        def run_transaction_with_retry(txn_func, session):  # type: ignore[no-redef]
```

Comment:
Here and below: We can't have type ignore comments in any of these examples otherwise they will get published to the docs. For example: https://docs.mongodb.com/manual/core/transactions-in-applications/#example-1

Is there another way? What if we `del` the old method beforehand?

Reply:
That didn't work, I added `--disable-error-code no-redef` to the options instead.